app/services/api/v1/data/emission_pathways/filter_columns.rb

```diff
@@ -48,10 +44,6 @@ def select_columns_map
                 column: 'composite_name',
                 alias: 'composite_name'
               },
-              {
-                column: 'unit',
```

Comment:
do we need to remove unit? I think it is displayed on the frontend?

Reply:
it was duplicated. there's another call to it further down.